lib/matplotlib/tests/test_widgets.py

```diff
@@ -1107,17 +1107,38 @@ def test_range_slider(orientation):
 
     slider.set_val((0.2, 0.6))
     assert_allclose(slider.val, (0.2, 0.6))
+    if orientation == "vertical":
+        hpositions = [h.get_ydata()[0] for h in slider._handles]
+    else:
+        hpositions = [h.get_xdata()[0] for h in slider._handles]
+    assert_allclose(hpositions, (0.2, 0.6))
```

Comment:
You might want to define a local function
```
def handle_positions(slider):
    if orientation == "vertical":
        return [h.get_ydata()[0] for h in slider._handles]
    else:
        return [h.get_xdata()[0] for h in slider._handles]
```
and simplify this and following checks to
```suggestion
    assert_allclose(handle_positions(slider), (0.2, 0.6))
```

Reply:
Great suggestion. I added the local function.